### PR TITLE
Add types for Pannellum

### PR DESCRIPTION
**Pull request by zhirzh**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/pannellum/pannellum-tests.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import ".";
```
Comment (peterblazejewicz) [inline types/pannellum/pannellum-tests.ts:1]: import package here (like when consuming in real code)

Comment (peterblazejewicz) [reply types/pannellum/pannellum-tests.ts:1]: or skip imports, if this is a global script

Comment (zhirzh) [reply types/pannellum/pannellum-tests.ts:1]: > import package here (like when consuming in real code)
> or skip imports, if this is a global script

Not sure if I understand. When using pannellum, we just import the library and it sets global object `window.pannellum` - the lib doesn't export anything.

https://github.com/mpetroff/pannellum/blob/2.5.6/src/js/pannellum.js#L24

Comment (peterblazejewicz) [reply types/pannellum/pannellum-tests.ts:1]: agree, but here one should import the module under test same way as expected when released, so tests cannot use file based imports. Use module name in the import, or skip import, this is global script, e.g.:
```suggestion
```

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@zhirzh thanks!

**Comment by zhirzh** [comment]
Ready to merge

**Pull request merged**